README.md

```diff
@@ -1 +1,21 @@
 ### ReadMe Test
+def mod_func(a,b):
+  return a%b
+
+def addition_func(a,b):
+  return a+b
+
+def mult_func(a,b):
+  c=a*b
+  return c
+
+
+a=51
```

Comment:
This line assigns the value 51 to the variable 'a'. No issues or improvements needed.